quickwit/quickwit-search/src/collector.rs

```diff
@@ -154,25 +154,53 @@ impl SortingFieldExtractorComponent {
     ///
     /// The function returns None if the sort key is a fast field, for which we have no value
     /// for the given doc_id, or we sort by DocId.
+    #[inline]
```

Comment:
The comment of the function and the name of the function need to be changed.
This function is now only return the u64 value.

Can we introduce another type for this?

Reply:
`SortingFieldExtractorComponent::Score` is quite a pain of course. I'm ok to treat it as u64 using tantivy's `u64_to_f64` if that helps making the other code path faster / simpler.

Reply:
I switched to u64, the main advantage is having a simpler compare function. It's additionally ~10% faster on average

Reply:
awesome.